index.html

```diff
@@ -3,449 +3,295 @@
-          dimensions of an image. Each keyword is either an <a>ASCII
-          case-insensitive</a> match for the <a>string</a> "<a data-lt=
-          "any link size">any</a>", or a value that consists of two <a>valid
-          non-negative integers</a> that do not have a leading U+0030 DIGIT
-          ZERO (0) character and that are separated by a single U+0078 LATIN
-          SMALL LETTER X or U+0058 LATIN CAPITAL LETTER X character. The
-          keywords represent icon sizes in raw pixels (as opposed to CSS
-          pixels). When multiple <a>ImageResource</a>s are available, a user
-          agent MAY use the value to decide which image is most suitable for a
-          display context (and ignore any that are inappropriate).
+          The {{ImageResource/sizes}} member is equivalent to a [^link^]
+          element's [^link/sizes^] attribute, and is processed in the same
+          manner.
         </p>
-        <div class="note">
-          The <a>sizes</a> member allows for multiple space-separated size
-          values in order to accommodate image formats (such as ICO) that can
-          act as a container for multiple images of varying dimensions.
-        </div>
         <p>
-          The steps for <dfn>processing the <code>sizes</code> member of an
-          image</dfn> are given by the following algorithm. The algorithm takes
-          an <a>ImageResource</a> <var>image</var>. This algorithm will return
-          a set.
+          When multiple {{ImageResource}} dictionaries are available, a user
+          agent MAY use the {{ImageResource/sizes}} value to decide which image
+          is most suitable for a display context (and ignore any that are
+          inappropriate).
         </p>
-        <ol>
-          <li>Let <var>set</var> be an empty <a>set</a>.
-          </li>
-          <li>Let <var>value</var> be <var>image.sizes</var>.
-          </li>
-          <li>If <a>Type</a>(<var>value</var>) it not String, return
-          <code>undefined</code>.
-          </li>
-          <li>Otherwise, parse <var>value</var> as if it was a [[HTML]]
-          {{HTMLLinkElement/sizes}} attribute and let <a>list</a>
-          <var>keywords</var> be the result.
-          </li>
-          <li>
-            <a>For each</a> <var>keyword</var> of <var>keywords</var>:
-            <ol>
-              <li>
-                <a data-lt="set-append">Append</a> <var>keyword</var>,
-                <a>lowercased</a>, to <var>set</var>.
-              </li>
-            </ol>
-          </li>
-          <li>Return <var>set</var>.
-          </li>
-        </ol>
+        <aside class="note">
+          When multiple {{ImageResource}} dictionaries are being used by an API
+          (e.g., in a WebIDL [=sequence=]), the specifications should indicate
+          if the order in which the dictionaries are passed is significant. As
+          a general rule, order is significant and "last one wins".
```

Comment:
Last one wins? I thought it was usually "first match wins."

Reply:
Still unsure which one is the right one... some specs do first, other do last. We might need to check a few specs.

Reply:
In HTML, `audio`, `picture`, and `video` all work this way with multiple `source` elements in play (which is pretty similar).

Reply:
With the sizes attribute, it's also first-match that wins. Are there any counter-examples, especially related to images?

Reply:
Not that I am aware of. I think we need to change it to "first match wins."

Reply:
Resolved in #20